CHANGELOG.md

```diff
@@ -187,7 +187,7 @@
     `a, b = #XCHG(a, b);` to allow swapping the contents of two operands.
   ([PR #678](https://github.com/jasmin-lang/jasmin/pull/678)).
 
-- The Constant-Time security checker is now available as a separate `jazzct`
```

Comment:
Don’t change the changelog of already released stuff. Please make a new changelog entry.